pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+                  This plugin is responsible for packaging artifacts
+                  as OSGi bundles.  Please refer to
+                  http://felix.apache.org/site/apache-felix-maven-bundle-plugin-bnd.html
+                  for more information about how to use this plugin.
+                -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.2</version>
+    </parent>
+
+    <groupId>org.eclipse.ee4j.websocket</groupId>
+    <artifactId>websocket-parent</artifactId>
+    <packaging>pom</packaging>
+    <version>1.1.1-SNAPSHOT</version>
+    <name>WebSocket API</name>
+    <description>JSR 356: Java API for WebSocket</description>
+    <url>https://projects.eclipse.org/projects/ee4j.websocket</url>
+
+    <scm>
+        <connection>scm:git:https://github.com/eclipse-ee4j/websocket-api</connection>
+        <developerConnection>scm:git:ssh://git@github.com/eclipse-ee4j/websocket-api</developerConnection>
+        <url>https://github.com/eclipse-ee4j/websocket-api</url>
+        <tag>1.1</tag>
+    </scm>
+
+    <properties>
+        <api_package>javax.websocket</api_package>
+        <spec_version>1.1</spec_version>
+        <!--<spec_build>02</spec_build>-->
+        <new_spec_version>1.1</new_spec_version>
+        <spec_impl_version>1.1.1</spec_impl_version>
+        <packages.export>javax.websocket.*</packages.export>
+    </properties>
+
+    <modules>
+        <module>api</module>
+        <module>spec</module>
```

Comment:
This is compatible with the `api` module.
`api` requires JDK6+
We've had issues using JDK8 to target JDK6 bytecode.